frontend/new-cm-sg-ls-vm/cm_setup_0.1.sh

```diff
@@ -7,13 +7,17 @@
 yum -y update
 
 #Install and setup the Sumo Collector
-wget https://teradeploy.blob.core.windows.net/binaries/SC_1.0.zip -P /tmp/
-unzip -o /tmp/SC_1.0.zip
-rpm -i sumo/SumoCollector-19.182-25.x86_64.rpm
-cp sumo/user.properties /opt/SumoCollector/config/
-cp sumo/sumo_cm_vm.json /opt/SumoCollector/config/
+mkdir sumo
```

Comment:
This should probably be an absolute path (in /tmp, preferably) rather than just creating a directory relative to the working directory of this script.